packages/accounts/src/utils.ts

```diff
@@ -12,8 +12,10 @@ function parseJsonFromRawResponse(response: Uint8Array): any {
     return JSON.parse(Buffer.from(response).toString());
 }
 
-function bytesJsonStringify(input: any): Buffer {
-    return Buffer.from(JSON.stringify(input));
+export function stringifyJsonOrBytes(args: any): Buffer {
```

Comment:
This method is already exported from `@near-js/transactions`, please import it from there and remove `bytesJsonStringify`.